### PR TITLE
Refactor(inji-472): refactoring api usage, fix warnings, and fix fabicon ui issue

### PR DESCRIPTION
**Pull request by vijay151096**
**Issues Fixed as a part of this PR :**

1.  Make the FAB Icon pressable in all places. 
2. Masking the UIN/VID in the VC Details Page.
3. Introduced Padding at the bottom of the MyVCs page to make the FAB pressable in all device screen sizes. 
4. Moved all the API's to api.ts irrespective of cached.
5. Fixed IOS build issue for taking the inputted host endpoints.
6. Removed the package.json entry for telemetry-sdk

### CODE REVIEW

**Review by vharsh — APPROVED** [review]
lgtm

File: components/VC/MosipVCItem/MosipVCItem.tsx [hunk 1]
```diff
@@ -73,7 +73,6 @@ export const MosipVCItem: React.FC<
               vcMetadata={props.vcMetadata}
               verifiableCredential={verifiableCredential}
               emptyWalletBindingId={emptyWalletBindingId}
-              onPress={() => props.onPress(service)}
```
Comment (swatigoel) [inline components/VC/MosipVCItem/MosipVCItem.tsx:76]: why this is removed?

File: components/VC/MosipVCItem/MosipVCItemDetails.tsx [hunk 2]
```diff
@@ -167,7 +168,7 @@ export const MosipVCItemDetails: React.FC<
                       weight="semibold"
                       size="smaller"
                       color={Theme.Colors.Details}>
-                      {uin}
+                      {getMaskedText(uin)}
```
Comment (swatigoel) [inline components/VC/MosipVCItem/MosipVCItemDetails.tsx:171]: In details, are we expecting to show masked data?

File: screens/Home/HomeScreen.tsx [hunk 3]
```diff
@@ -44,15 +45,15 @@ export const HomeScreen: React.FC<HomeRouteProps> = props => {
       <LinearGradient
         colors={Theme.Colors.gradientBtn}
         style={Theme.Styles.downloadFabIcon}>
-        <Button
-          testID="downloadIcon"
-          icon={plusIcon}
+        <Pressable
```
Comment (swatigoel) [inline screens/Home/HomeScreen.tsx:48]: We need to add testId to Fab icon.
As we are wrapping this in Pressable, just check whether we get it in appium for iOS and Android both.
If not, we might need to add `accessible={false}`

**Review by swatigoel — CHANGES_REQUESTED** [review]

File: components/ui/themes/DefaultTheme.ts [hunk 4]
```diff
@@ -529,14 +529,28 @@ export const DefaultTheme = {
...
       padding: 10,
-      backgroundColor: Colors.Orange,
       shadowColor: '#000',
       shadowOpacity: 0.4,
       elevation: 5,
       position: 'absolute',
       bottom: Dimensions.get('window').width * 0.1,
       right: Dimensions.get('window').width * 0.1,
     },
+    fabIconNormal: {
```
Comment (KiruthikaJeyashankar) [inline components/ui/themes/DefaultTheme.ts:539]: Can we download  / plus word also here to be more explicit about this icon use-case (download card button)? like downloadFabIconNormal

**Review by KiruthikaJeyashankar — APPROVED** [review]

File: components/VC/MosipVCItem/MosipVCItemDetails.tsx (hunk 2 above)
Comment (vijay151096) [reply components/VC/MosipVCItem/MosipVCItemDetails.tsx:171]: checked with @kratitva, new ui is coming up which will involve lot of ui refactoring, so for now, lets go with the existing approach. so removing this.

File: components/VC/MosipVCItem/MosipVCItem.tsx (hunk 1 above)
Comment (vijay151096) [reply components/VC/MosipVCItem/MosipVCItem.tsx:76]: onPress is not being used anywhere in the activation, Hence removing it.

File: components/ui/themes/DefaultTheme.ts (hunk 4 above)
Comment (vijay151096) [reply components/ui/themes/DefaultTheme.ts:539]: Updated it.

File: screens/Home/HomeScreen.tsx (hunk 3 above)
Comment (vijay151096) [reply screens/Home/HomeScreen.tsx:48]: added the testid for it.

**Review by swatigoel — APPROVED** [review]

**Pull request merged**